bundles/CoreBundle/Migrations/Version20201008101817.php

```diff
@@ -28,7 +29,9 @@ final class Version20201008101817 extends AbstractMigration
      */
     public function up(Schema $schema): void
     {
-        $this->addSql('DROP VIEW IF EXISTS documents_editables;');
+        try {
+            $this->addSql('DROP VIEW IF EXISTS documents_editables;');
+        } catch (Exception) {}
```

Comment:
Maybe write a line to the console that something happened here?

```suggestion
        } catch (Exception $e) {
            $this->write("Could not drop view: " . $e->getMessage());
        }
```